error.go

```diff
@@ -191,7 +242,7 @@ func (e *PermissionError) Error() string {
 // reason.
 type CardError struct {
 	stripeErr   *Error
-	DeclineCode string `json:"decline_code,omitempty"`
+	DeclineCode DeclineCode `json:"decline_code,omitempty"`
```

Comment:
Would you mind changing this line to something like:

``` go
	// DeclineCode is a code indicating a card issuer's reason for declining a
	// card (if they provided one).
	//
	// TODO: This field should be of type DeclineCode (as defined above), but
	// we've let it as a string for now for backwards compatibility. Change its
	// type on the library's next major version.
	DeclineCode string `json:"decline_code,omitempty"`
}
```

Reply:
hm good catch. what do you think about using type alias (ie., `type DeclineCode = string`)

Reply:
Good question. I'd still say we should just put a `string`: the long term intent is that `DeclineCode` not be a type alias, so I'd say we just make it the right thing now, and then keep it that way.

Admittedly, we have to change something either way on the next major, but omitting the type alias in favor of a simple `string` makes looking at what the field's definition more clear in my mind.